src/Content/Features.php

```diff
@@ -0,0 +1,148 @@
+			),
+
+			// Network tabs
+			'net_tabs' => array(
+				t('Network Tabs'),
+				array('personal_tab',	t('Network Personal Tab'),		t('Enable tab to display only Network posts that you\'ve interacted on'), false, Config::get('feature_lock','personal_tab', false)),
+				array('new_tab',	t('Network New Tab'),			t('Enable tab to display only new Network posts (from the last 12 hours)'), false, Config::get('feature_lock','new_tab', false)),
+				array('link_tab',	t('Network Shared Links Tab'),		t('Enable tab to display only Network posts with links in them'), false, Config::get('feature_lock','link_tab', false)),
+			),
+
+			// Item tools
+			'tools' => array(
+				t('Post/Comment Tools'),
+				array('multi_delete',	t('Multiple Deletion'),			t('Select and delete multiple posts/comments at once'), false, Config::get('feature_lock','multi_delete', false)),
+				array('edit_posts',	t('Edit Sent Posts'),			t('Edit and correct posts and comments after sending'), false, Config::get('feature_lock','edit_posts', false)),
+				array('commtag',	t('Tagging'),				t('Ability to tag existing posts'), false, Config::get('feature_lock','commtag', false)),
+				array('categories',	t('Post Categories'),			t('Add categories to your posts'), false, Config::get('feature_lock','categories', false)),
+				array('filing',		t('Saved Folders'),			t('Ability to file posts under folders'), false, Config::get('feature_lock','filing', false)),
+				array('dislike',	t('Dislike Posts'),			t('Ability to dislike posts/comments'), false, Config::get('feature_lock','dislike', false)),
+				array('star_posts',	t('Star Posts'),			t('Ability to mark special posts with a star indicator'), false, Config::get('feature_lock','star_posts', false)),
+				array('ignore_posts',	t('Mute Post Notifications'),		t('Ability to mute notifications for a thread'), false, Config::get('feature_lock','ignore_posts', false)),
+			),
+
+			// Advanced Profile Settings
+			'advanced_profile' => array(
+				t('Advanced Profile Settings'),
+				array('forumlist_profile', t('List Forums'),			t('Show visitors public community forums at the Advanced Profile Page'), false, Config::get('feature_lock','forumlist_profile', false)),
+				array('tagadelic',	t('Tag Cloud'),				t('Provide a personal tag cloud on your profile page'), false, Config::get('feature_lock', 'tagadelic', false)),
+			),
+		);
+
+		// removed any locked features and remove the entire category if this makes it empty
+
+		if ($filtered) {
+			foreach ($arr as $k => $x) {
+				$has_items = false;
+				$kquantity = count($arr[$k]);
+				for ($y = 0; $y < $kquantity; $y ++) {
+					if (is_array($arr[$k][$y])) {
+						if ($arr[$k][$y][4] === false) {
+							$has_items = true;
+						} else {
+							unset($arr[$k][$y]);
+						}
+					}
+				}
+				if (! $has_items) {
+					unset($arr[$k]);
+				}
+			}
+		}
+
+		call_hooks('get',$arr);
```

Comment:
This file is supposed to require include/plugin.php to call this function.
Also can you please add a space after the comma?